recursive_count_th/count_th.py

```diff
@@ -3,8 +3,25 @@
 Your function should return a count of how many occurences of ***"th"*** occur within `word`. Case matters.
 Your function must utilize recursion. It cannot contain any loops.
 '''
+arr = []
 def count_th(word):
-    
-    # TBC
-    
-    pass
+    count = 0
+    if 'th' not in word:
+        global arr
+        count = len(arr)
+        arr = []
+        print(count)
+        return count
+    else:
+        arr.append('th')
+        word = word.replace('th', 'a', 1)
+
+    return count_th(word)
```

Comment:
You are recursively calling `count_th` here but in an incorrect fashion. You should look back at this problem and see if you can figure out a way to get the function to return the correct count without using `len(arr)`.